docs/modules/ROOT/pages/explanations/disable-billing.adoc

```diff
@@ -0,0 +1,23 @@
+= How Disabling Billing Works
+
+For services that depend on other services, like Keycloak -> PostgreSQL, we don't want to bill the customer with both services.
+
+In order to achieve that, we need a mechanism with which we can disable the billing for any given service, but in such a way that it's not possible for the user to manipulate it.
+
+As all settings between the claim and the composite are synced, we can't expose any configuration regarding this in the claim. Otherwise, the user would be able to simply disable the billing for any service. In order to prevent that, the configuration to disable billing should be passed via means that are inaccessible for the end-user but accessible to the composition functions.
+
+There's already a namespace for such things: `appcat-control`. That's a namespace specifically used for things that the end-user should not be able to see or manipulate.
```

Comment:
Don't we already have `syn-appcat `namespace? Is it the same namespace?

Reply:
`syn-appcat` is where appcat itself is deployed. `appcat-control` is where some of the jobs run with clusterrolebindings, that a customer should not be able to access at all. Mostly restore and maintenance jobs.